fabric-convention-tags-v1/src/client/java/net/fabricmc/fabric/impl/tag/convention/client/ConventionLogWarnings.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.tag.convention.client;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Properties;
+
+import it.unimi.dsi.fastutil.objects.ObjectArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.minecraft.client.resource.language.I18n;
+import net.minecraft.item.Item;
+import net.minecraft.registry.DynamicRegistryManager;
+import net.minecraft.registry.Registry;
+import net.minecraft.registry.RegistryKeys;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents;
+import net.fabricmc.fabric.api.tag.convention.v1.TagUtil;
+import net.fabricmc.loader.api.FabricLoader;
+
+public class ConventionLogWarnings implements ClientModInitializer {
```

Comment:
Why is this client-only?

Tags are mostly a server-side concept, which means server-exclusive mods won't have the warning.

Reply:
The class also has the warnings for the `c` namespace, not just untranslated tags.

Reply:
The  translation stuff is client sided and I was too lazy to split it out from the other log warning. I'll take a stab at separating maybe tonight or tomorrow.

Reply:
Split up now. The translation key warning remains in client side but the legacy tag warning now exists in main init. Should be much better. Also split the config stuff into its own class for cleaner organizing